vllm/datadog_checks/vllm/check.py

```diff
@@ -0,0 +1,52 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from datadog_checks.base import AgentCheck, OpenMetricsBaseCheckV2  # noqa: F401
+
+from .metrics import METRIC_MAP, RENAME_LABELS_MAP
+
+DEFAULT_VERSION_ENDPOINT = "/version"
+DEFAULT_HEALTH_ENDPOINT = "/health"
```

Comment:
Where are these used?

Reply:
Not used anymore since the version is added explicitly and we're no longer submitting a service check for the `/health` endpoint. Ilia just may have forgotten to remove them 👍

Reply:
good catch, removing!